apps/openmw/mwmechanics/character.cpp

```diff
@@ -2372,7 +2372,7 @@ void CharacterController::update(float duration, bool animationOnly)
     moved.x() *= scale;
     moved.y() *= scale;
 
-    if(mPtr.getClass().isNpc())
+    if (mPtr.getClass().isNpc() && !Settings::Manager::getBool("normalise race speed", "Game"))
```

Comment:
In there a reason to retrieve the setting every frame for every NPC?

Reply:
I see what you mean. Adjusted it.